src/Tribe/Rewrite.php

```diff
@@ -456,9 +456,9 @@ protected function get_matcher_to_query_var_map() {
 		 * @param  array  array of the current matchers for query vars.
 		 * @param  self   $rewrite
 		 */
-		$matchers = apply_filters( 'tribe_events_rewrite_matchers_to_query_vars_map', $matchers, $this );
+		$map = apply_filters( 'tribe_events_rewrite_matchers_to_query_vars_map', $map, $this );
```

Comment:
* :no_entry_sign: Spaces must be used to indent lines; tabs are not allowed
* :warning: Line exceeds 80 characters; contains 95 characters

<sub>hash: 58e0a7771d1c4f0441dce6b5f912da9c</sub>